apps/plotly_examples/patches/@griffel+react+1.5.30.patch

```diff
@@ -0,0 +1,273 @@
+diff --git a/node_modules/@griffel/react/RendererContext.cjs.js b/node_modules/@griffel/react/RendererContext.cjs.js
+index 20502f5..7f99605 100644
+--- a/node_modules/@griffel/react/RendererContext.cjs.js
++++ b/node_modules/@griffel/react/RendererContext.cjs.js
```

Comment:
How are we using this patch

Reply:
I dont see patch-package script anywhere. We dont need to have a full patch. We just need to find replace the SECRET INTERNAL statement with the correct one

Reply:
still not seeing the script